stable/rabbitmq-ha/templates/alerts.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if and .Values.prometheus.exporter.enabled .Values.prometheus.operator.enabled .Values.prometheus.operator.alerts.enabled }}
+{{ if and .Values.prometheus.operator.enabled .Values.prometheus.operator.alerts.enabled }}
```

Comment:
Should probably mirror the logic in servicemonitor.yaml:
`if and (or .Values.prometheus.exporter.enabled .Values.rabbitmqPrometheusPlugin.enabled) .Values.prometheus.operator.enabled .Values.prometheus.operator.alerts.enabled }}`

Reply:
The `servicemonitor.yaml` file is to install monitoring target, so it make sense that it tests `prometheus.exporter.enabled` and `rabbitmqPrometheusPlugin.enabled` values to determine scrapping endpoint to declare.
But the `alerts.yaml` is to install alert rules : we just need to known if rules must be installed via `prometheus.operator.alerts.enabled` value, no matter which target is installed. For user it will not be clear that rules are installed depending of four values.

Reply:
Makes sense. Thanks for clarifying. Can you merge from master and fix version?